examples/third_party/autotools/BUILD.automake.bazel

```diff
@@ -9,20 +9,20 @@ filegroup(
 
 configure_make(
     name = "automake",
-    build_data = [
-        "@autoconf",
-        "@m4",
-    ],
     env = {
-        "AC_MACRODIR": "$$EXT_BUILD_DEPS$$/bin/autoconf/share/autoconf",
```

Comment:
The fact that this style of env bar doesn't work anymore seems like a regression..